articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -41,6 +42,8 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     }
     ```
 
+You can find an example Django application in the sample Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).
+
```

Comment:
Might want to link to [this](https://github.com/Azure-Samples/azure-monitor-opencensus-python/blob/master/docs_samples/02_python_app_insights_dependencies.md#configuring-django-applications) instead of just the root folder.